streams/examples/src/main/java/org/apache/kafka/streams/examples/wordcount/WordCountTransformerDemo.java

```diff
@@ -153,8 +154,8 @@ public void run() {
             streams.start();
             latch.await();
         } catch (final Throwable e) {
-            System.exit(1);
+            Exit.exit(1);
         }
-        System.exit(0);
+        Exit.exit(0);
```

Comment:
I think it was intentional not to use an internal class in demos that others may copy.

Reply:
Ok, I can back this change out of the demos. I'd taken an "agricultural fire" approach to this, since it seemed like the error was coming from a source I couldn't identify. Namely, the reported error is from the "unitTest" task, whereas the only test that would have called "Sytem.exit" I could find was in the JUnit IntegrationTest category.